cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -338,7 +338,8 @@ describe("Timeline", () => {
 
             // Exclude timestamp from snapshot
             const percyCSS =
-                ".mx_RoomView_body .mx_EventTile_info .mx_MessageTimestamp { visibility: hidden !important; }";
+                ".mx_RoomView_body .mx_EventTile .mx_EventTile_line .mx_MessageTimestamp " +
+                "{ visibility: hidden !important; }";
             cy.get(".mx_EventTile_last").percySnapshotElement("URL Preview", {
```

Comment:
> which checks a normal EventLine instead of an info EventLine.

Did you want to change the selector in L343 instead by chance? L341 should really just be `.mx_MessageTimestamp { visibility: hidden !important; }`

Reply:
> L341 should really just be .mx_MessageTimestamp { visibility: hidden !important; }

Exactly, this should work. My intent was to change the incorrect selector (`.mx_EventTile_info` should be replaced as this is not an info event tile).